docs/product/guidelines/javascript.html

```diff
@@ -5,40 +5,37 @@
 ---
 
 <section class="stacks-section">
-
     {% header h2 | Including the Stacks JavaScript %}
 
-    <p class="stacks-copy">While Stacks is first and foremost a CSS library, it also provides commonly used functionality for some components via JavaScript code. Using this functionality is optional. If you only need the styling parts of Stack, you’re free to ignore the provided JavaScript. The converse is not true: The JavaScript components work under the assumption that the Stacks CSS is available.</p>
+    <p class="stacks-copy">While Stacks is first and foremost a CSS library, it also provides commonly used functionality for some components via JavaScript. This functionality is optional. If you only need the styling parts of Stack, you’re free to ignore the provided JavaScript. The converse is not true: The JavaScript components work under the assumption that the Stacks CSS is available.</p>
 
-    <p class="stacks-copy">In order to use the JavaScript components, do the following.</p>
+    <p class="stacks-copy">Stacks JavaScript is currently included within various Stack Overflow projects automatically. If you’re working on a Stack Overflow project, chances are it’s already available for you! If not, reach out to us and we’ll work on getting it setup.</p>
```

Comment:
This isn't true. Shouldn't we wait before claiming this until it's actually the case?